ash/src/extensions/ext/calibrated_timestamps.rs

```diff
@@ -55,15 +41,54 @@ impl CalibratedTimestamps {
         .result_with_success((timestamps, max_deviation))
     }
 
-    pub const NAME: &'static CStr = vk::ExtCalibratedTimestampsFn::NAME;
+    pub const NAME: &'static CStr = vk::ExtCalibratedTimestampsDeviceFn::NAME;
```

Comment:
I've been contemplating keeping the original non-suffixed struct and leaving the two associated constants there...